README.md

```diff
@@ -8,435 +8,429 @@ ACL (Access Control List)
-        * [Removing ACE permissions](#removing-ace-permissions)
-        * [Same identity, multiple ACEs](#same-identity-multiple-aces)
-5. [Limitations - Known issues in acl](#limitations)
-6. [Development - Guide for contributing to the module](#development)
-
+5. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+6. [Limitations - Known issues in acl](#limitations)
+7. [Development - Guide for contributing to the module](#development)
 
 ##Overview
 
-The acl module uses Puppet to manage Access Control Lists on Windows.
+The acl module lets you use Puppet to manage Access Control Lists (ACLs) on Windows.
 
 ##Module Description
 
-The acl module adds a type and provider for managing permissions through Windows ACLs.
-
-ACLs typically contain a list of access control entries (ACEs). An ACE is a defined trustee (identity) with a set of rights. For each ACE, the ACL contains an allowed/denied status, as well as the ACE's inheritance and propagation strategy. The order of ACEs within the ACL determines what allowed/denied status is applied first.
+Windows uses Access Control Lists (ACLs) to store permissions information. The acl module adds a type and provider for managing that information through Puppet.
 
 ##Setup
 
-The best way to install this module is by executing the following command on your puppet master or local Puppet install:
+Install this module with the following command:
 
-    $ puppet module install [--modulepath <path>] puppetlabs/acl
 
-The above command also includes the optional argument to specify your puppet master's `modulepath` as the location to install the module.
+~~~
+$ puppet module install [--modulepath <path>] puppetlabs/acl
+~~~
 
-###Beginning with acl
+The above command also includes the optional argument to specify your Puppet master's `modulepath` as the location to install the module.
 
-Each ACE has one of four possible statuses and the order you list the ACEs in is important for which status is applied first. The allow/deny types available are: 'explicit deny', 'explicit allow', 'inherited deny', and 'inherited allow'.
+###Beginning with acl
 
-You cannot specify inherited ACEs in a manifest; you can only specify whether to allow upstream inheritance to flow into the managed target location (known as security descriptor). Please ensure your modeled resources follow this order or Windows will complain. The `acl` type **does not** enforce or complain about ACE order.
+For a basic implementation of the acl module, provide a target ACL and at least one permission:
 
-For a basic implementation of the acl module, you need to provide the target and at least one permission (access control entry or ACE). It will default the other settings to sensible defaults.
 
-    acl { 'c:/tempperms':
-      permissions => [
-       { identity => 'Administrator', rights => ['full'] },
-       { identity => 'Users', rights => ['read','execute'] }
-     ],
-    }
+~~~
```

Comment:
They indicate a block of code (GitHub translates them to <pre> tags in the HTML.) The current README does the same thing using four-space indentations, but the tildes are what we use in most of our other READMEs so I'm switching over for consistency. I also like the tildes because they're easier to eyeball-check, and I believe they're supported by a longer list of other Markdown interpreters.